.github/workflows/ci_dockers.yml

```diff
@@ -10,6 +10,11 @@ on:  # Trigger the workflow on push or pull request, but only for the master bra
 
 jobs:
   build-Conda:
+    if: |
+      !contains(github.event.commits.*.message, '[skip ci]') ||
```

Comment:
@williamFalcon any preferences for patterns?
I would stick just with tags in brackets if it plain text it could be an accident and so confusion for user with missing builds...

Reply:
so [ci skip] or [skip ci] or both?